custom_components/sat/manufacturer.py

```
@@ -1,54 +1,63 @@
-    def name(self) -> str:
+    def friendly_name(self) -> str:
         pass
 
 
 class ManufacturerFactory:
     @staticmethod
     def resolve_by_name(name: str) -> Optional[Manufacturer]:
         """Resolve a Manufacturer instance by its name."""
-        manufacturer = MANUFACTURERS.get(name)
-        if not manufacturer:
+        if not (member_id := MANUFACTURERS.get(name)):
```

Ruff (0.8.2)

Local variable `member_id` is assigned to but never used

(F841)